### PR TITLE
Add useful plugins

### LINKED ISSUE #9: Add favicon to website
Website is currently missing favicon

### LINKED ISSUE #10: Add sitemap to website
Website is missing sitemap:
https://guide.memo.bank/sitemap.xml

### LINKED ISSUE #11: Change website language
Current website language is set to English in html
`<html lang="en-US">`

### PR DESCRIPTION
**Pull request by bboulesteix**
This pull request:  closes #9, closes #10, and closes #11.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -39,15 +66,24 @@ kramdown:
   auto_ids: true
   syntax_highlighter: rouge
 
+# Permalin
```
Comment (dirtyhenry) [inline _config.yml]: Typo, missing a k.

File: Gemfile [hunk 2]
```diff
@@ -34,4 +38,5 @@ gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 # do not have a Java counterpart.
 gem "http_parser.rb", "~> 0.6.0", :platforms => [:jruby]
 
-gem "webrick", "~> 1.7"
+# Add webrick to fix a bug with Ruby version 3.0.0 or higher
+gem "webrick", "~> 1.7"
```
Comment (dirtyhenry) [inline Gemfile]: Could you add a blank line at the end of this file? 🙏

**Review by dirtyhenry — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (bboulesteix) [reply Gemfile]: Yes, will do. Thanks @dirtyhenry. 👍

File: _config.yml (hunk 1 above)
Comment (bboulesteix) [reply _config.yml]: Oh thanks for pointing that typo out. 👍

**Review by dirtyhenry — APPROVED** [review]

**Pull request merged**